### DISCUSSION TITLE
Model not showing but no error

### DISCUSSION THREAD

**Discussion by rickvana** [google/model-viewer#3010] [General]
Hello everyone,

I'm trying to show a model on my webpage. I got the model (.gltf) file, I put everything in but it's not showing, and also not showing any errors. 

![afbeelding_2021-12-02_132529](https://user-images.githubusercontent.com/61792858/144421538-5739e626-2f3d-4dab-9341-244f924d6a3d.png)
![afbeelding_2021-12-02_132549](https://user-images.githubusercontent.com/61792858/144421584-550bfa9a-7d79-45c7-9277-612917982e09.png)

**Comment by elalish** [comment]
We'll need a link to  your live site to see what's wrong. Also `ios-src` does not accept gltfs, though this is not the source of your current problem. Also, the legacy version has been deprecated, so  it's no longer  needed.

**Reply by rickvana** [reply]
Hey! The website is currently not live yet, I will implement these changes and see what the output is.

EDIT: Still doesn't work
